src/lib/datepicker/datepicker.spec.ts

```diff
@@ -1632,6 +1632,53 @@ describe('MatDatepicker', () => {
 
       expect(document.querySelector('mat-calendar-header')).toBeTruthy();
     }));
+
+    it('should find the custom element', fakeAsync(() => {
+        testComponent.datepicker.open();
+        fixture.detectChanges();
+        flush();
+        fixture.detectChanges();
+
+        expect(document.querySelector('.custom-element')).toBeTruthy();
+    }));
+  });
+
+  describe('datepicker with custom header using ng-content', () => {
```

Comment:
I'm not sure whether we need a test for this. It's basically testing if `ng-content` works which isn't something Material is responsible for.

Reply:
I agree with this comment for the `datepicker with custom header using ng-content` suite I added and will remove.

However I think the additional test that checks that the custom element is added is useful for the `datepicker with custom header` suite. Prior to my change to DatepickerWithCustomHeader a custom header component was not in fact being used by that test suite.